src/main/resources/db/migration/V0001__Inin_migration(animal_types).sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `animal_types` (
```

Comment:
Why we divided sql queries to more one file ?

Reply:
I guess that for migration and develop DB must be more than  one file.Step by stap. If I am wrong, I will make one file.

Reply:
done